src/components/motebehov/MotebehovSide.tsx

```diff
@@ -128,20 +123,23 @@ const MotebehovSide = ({ fnr }: MotebehovSideProps) => {
     return <Feilmelding />;
   }
   return (
-    <Motebehov
-      fnr={fnr}
-      ledereData={ledereData}
-      ledereUtenInnsendtMotebehov={ledereUtenInnsendtMotebehov}
-      motebehovListe={activeMotebehovSvar}
-      sykmeldt={sykmeldt}
-      motebehovListeMedJaSvarTilOppgavebehandling={
-        motebehovListeMedJaSvarTilOppgavebehandling
-      }
-      veilederinfo={veilederinfo}
-      aktiveDialoger={aktiveDialoger}
-      oppfolgingstilfelleperioder={oppfolgingstilfelleperioder}
-      sykmeldinger={sykmeldinger}
-    />
+    sykmeldt &&
```

Comment:
Her kan det muligens bli returnert noe tull dersom ikke sykmeldt og veilederinfo er satt? Du kan jo enten gjøre noe ala (symeldt && veilederinfo) ? <Motebehov ... /> : </>, slik at du returnerer en fallback wrapper.

Eller bare droppe sjekkene, og sørge for at komponenten rendrer som den skal selv om dataene lastes (Enten med spinner, stensilering eller venting osv)

Reply:
..siden er komponent må returnere et eller annet, mens her blir det vel enten Motebehov eller ingenting

Reply:
Fikset slik at mulige undefined verdier håndteres.